openclsim/core.py

```diff
@@ -800,36 +802,40 @@ def check_weather_restriction(self, location, event):
         elif event not in self.work_restrictions[location.name].keys():
             self.calc_work_restrictions(location)
 
-        for criterion in sorted(self.work_restrictions[location.name].keys()):
-            ranges = self.work_restrictions[location.name][criterion]
+        for event_name in sorted(self.work_restrictions[location.name].keys()):
```

Comment:
Maybe this should be extended, if we are going to "events", just the name might not do it. It could for instance be that an event has multiple conditions (Hs < x if Tp > y, or Hs< x if U10 > y). 

What do you think about this?

Reply:
I actually chose this naming convention, because in this part of the code what used to be called 'criterion' actually represents criterion.event_name. As in the code just above this peace of code the 'criterion' name is used for a criterion object, this didn't consequent. So I merely tried to add a bit of consistency.

I agree that if you want to specify multiple conditions for an event in one criterion, the code should be extended. It is already possible however to specify individual conditions (individual criterions) for the same event, so then this would not be a problem. 

If we do however want to specify multiple conditions in one criterion, then I agree that the code should be extended. However, the WorkabilityCriterion class should then also be extended. I think this would be outside of the scope of this PR. 

Do you agree?

Reply:
@uijl A quick look at the code made me realise that it is actually is not possible to specify multiple individual conditions for the same event. Therefore I think your suggestion really makes sense!

I still feel that this would have to become a seperate PR (as stated in the last alinea of my previous comment). 

What do you think?

Reply:
I am currently working on the multiple individual criterions for a single event. 
This should have been working already, I messed it up somewhere.